recipes/torchhd/LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
This license is already included in the tarball that you can download from PyPI so you can remove it.